next.config.js

```diff
@@ -7,6 +7,9 @@ const withTM = require('next-transpile-modules')([
 ])
 
 module.exports = withPlugins([withTM, withImages, withPWA], {
+  images: {
```

Comment:
could you add a comment beside this to say we can remove this once next.js is updated

Reply:
Fixed